server/services/nextcloud-talk/lib/bot/bot.startPolling.js

```diff
@@ -22,7 +22,48 @@ async function startPolling(userId) {
     userId,
   );
 
+  if (
+    !this.bots[userId].NEXTCLOUD_URL ||
+    !this.bots[userId].NEXTCLOUD_BOT_USERNAME ||
+    !this.bots[userId].NEXTCLOUD_BOT_PASSWORD
+  ) {
+    logger.debug(`Cancel Nextcloud Talk bot starting for user: ${userId}, missing configuration`);
+    delete this.bots[userId];
```

Comment:
I don't think this will remove the listener, you need to call removeListener

Reply:
That's why I moved the addListener at the end of this function
https://github.com/GladysAssistant/Gladys/pull/1832/files#diff-c305aedfe417914247d3eeaa16a6c8d999e24c20c0c8f3a12ed3e6abbff83ad1R66

Reply:
But if startPolling is called again, will it create a second listener?

Reply:
Indeed, did not think about that case, should be better now

Reply:
Thanks for fixing!